setup.py

```diff
@@ -34,15 +34,16 @@ def load_about():
     },
     license="AGPLv3",
     author="Sofiane Bebert",
-    description="enterprisecatalog plugin for Tutor",
+    description="enterprise-catalog plugin for Tutor",
     long_description=load_readme(),
```

Comment:
If we want to replace RST with MD, we should change the README extension in line 9.

Reply:
Done.